classes/Link.php

```diff
@@ -494,35 +494,46 @@ public function getAdminLink($controller, $with_token = true, $sfRouteParams = a
 
         $params = $with_token ? array('token' => Tools::getAdminTokenLite($controller)) : array();
 
+        // Even if URL rewriting is not enabled, the page handled by Symfony must work !
+        // For that, we add an 'index.php' in the URL before the route
+        global $kernel; // sf kernel
+        $sfRouter = $kernel->getContainer()->get('router');
```

Comment:
what happens if Symfony kernel is not available at this moment ? For instance on installation process ?